subprojects/js/CMakeLists.txt

```diff
@@ -14,9 +14,10 @@ set(CMAKE_CXX_FLAGS "-std=c++11 -O3 -DOJPH_DISABLE_INTEL_SIMD")
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/../html)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/../html)
 
-file(GLOB SOURCES "src/ojph_wrapper.cpp" "../../src/core/coding/*.cpp" "../../src/core/others/*.cpp" "../../src/core/codestream/*.cpp" "../../src/core/transform/ojph_colour.cpp" "../../src/core/transform/ojph_transform.cpp")
+file(GLOB SOURCES "src/ojph_wrapper.cpp" "../../src/core/coding/*.cpp" "../../src/core/others/*.cpp" "../../src/core/codestream/*.cpp" "../../src/core/transform/ojph_colour.cpp" "../../src/core/transform/ojph_transform.cpp" "../../src/apps/others/ojph_img_io.cpp")
 add_executable(libopenjph ${SOURCES})
 include_directories(../../src/core/common)
+include_directories(../../src/apps/common)
```

Comment:
In my mind, the ojph_img_io.cpp is part of the executable, not the library.

Reply:
Right - I think this was a hack to support some JS hacking I was doing

Reply:
That's fine. I will not add it.